### DISCUSSION TITLE
Turning off FreeSync ingame makes game freeze

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#21407] [Q&A] [answered]
### Type

Game behaviour

### Bug description

There's actually 2 problems:
 1) Whenever I turn off FreeSync on my monitor the game freezes and windows doesn't detect a crash so I have to close it manually.
 
 2) Whenever I choose the Vsync option ingame it does not detect that my monitor is 144hz and instead locks at 60hz (This happens when FreeSync is off and when FreeSync is on)

I have already seen #8963 but none of the solutions work for me.

Lazer Version: 2022.1123.0-lazer
GPU: RX570 
Driver Version: AMD Adrenalin 22.20.27.07-221020a-384804C
Windows 21H2

### Version

2022.1123.0

### Logs

[network.log](https://github.com/ppy/osu/files/10085366/network.log)
[performance.log](https://github.com/ppy/osu/files/10085367/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10085368/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10085369/updater.log)
[database.log](https://github.com/ppy/osu/files/10085370/database.log)
[input.log](https://github.com/ppy/osu/files/10085371/input.log)

I turned off FreeSync at 14:26 and the game stopped responding to my inputs but music kept playing

**Comment by bdach** [comment, marked as answer]
Are your graphics drivers up to date? Or alternatively, is this a recent issue that popped up after a driver update?

**Reply by localcake** [reply]
Issue 2. is kind of random. Some weeks it happens and I just have to play with "2x refresh rate" despite all the tearing but since November started I haven't had this framerate detection issue. Maybe it has to do with driver updates and osu updates?

Issue 1. is brand new so I will try updating Windows and updating my drivers to the newest version and see what happens

**Reply by localcake** [reply]
After updating both Windows and AMD driver to the latest versions both problems seem to be fixed. I just hope that it doesn't come back again a few updates later
